src/shared-components/postCard/PostCard.tsx

```
@@ -422,7 +425,7 @@ export default function PostCard({ ...props }: InterfacePostCard): JSX.Element {
             <img src={props.image || UserDefault} alt={props.title} />
           ))}
         {props.video && (
-          <video controls className={styles.videoFullWidth}>
+          <video controls style={{ width: '100%' }}>
             <source src={props.video} type="video/mp4" />
           </video>
```

Biome (2.1.2)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)